automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/rstudio/RStudioSpec.scala

```diff
@@ -1,26 +1,26 @@
 package org.broadinstitute.dsde.workbench.leonardo.rstudio
 
 import org.broadinstitute.dsde.workbench.leonardo._
-import org.broadinstitute.dsde.workbench.leonardo.notebooks.NotebookTestUtils
+import org.openqa.selenium.{Keys}
 import org.scalatest.DoNotDiscover
-
-import scala.util.Try
-
 @DoNotDiscover
-class RStudioSpec extends RuntimeFixtureSpec with NotebookTestUtils {
+class RStudioSpec extends RuntimeFixtureSpec with RStudioTestUtils {
 
   override val toolDockerImage: Option[String] = Some(LeonardoConfig.Leonardo.rstudioBioconductorImage.imageUrl)
 
   "RStudioSpec" - {
 
     "should launch RStudio" in { runtimeFixture =>
-      withWebDriver { _ =>
-        // See this ticket for adding more comprehensive selenium tests for RStudio:
-        // https://broadworkbench.atlassian.net/browse/IA-697
-        val getResult = Try(RStudio.getApi(runtimeFixture.runtime.googleProject, runtimeFixture.runtime.clusterName))
-        getResult.isSuccess shouldBe true
-        getResult.get should include("unsupported_browser")
-        getResult.get should not include "ProxyException"
+      withWebDriver { implicit driver =>
+        withNewRStudio(runtimeFixture.runtime) { rstudioPage =>
+//          // See this ticket for adding more comprehensive selenium tests for RStudio:
+//          // https://broadworkbench.atlassian.net/browse/IA-697
+          Thread.sleep(10000)
+          rstudioPage.pressKeys("varA <- 1000")
```

Comment:
Awesome!! does this work?  :)

Reply:
Yep, looks like it does work. I do want to see if there's a more precise selector for the global environment table, but Keys does seem to work as intended

Reply:
More specifically, press keys works here because the console is already active when we get onto the page. There may be some more complexity in typing into other areas of the screen, but for now this should satisfy what we need